oneandone/src/main/java/org/apache/jclouds/oneandone/rest/OneAndOneApi.java

```diff
@@ -16,13 +16,11 @@
  */
 package org.apache.jclouds.oneandone.rest;
 
-import autovalue.shaded.com.google.common.common.annotations.Beta;
 import java.io.Closeable;
 import org.apache.jclouds.oneandone.rest.features.ImageApi;
 import org.apache.jclouds.oneandone.rest.features.ServerApi;
 import org.jclouds.rest.annotations.Delegate;
 
-@Beta
```

Comment:
No need to keep it; just commented in case you just removed it to make checkstyle happy :)
It is just an annotation to give information that the api is still in an early stage (which is redundant for providers in labs). There is no need to keep it.